apache-poi-3/src/main/java/com/baeldung/fastexcel/FastexcelHelper.java

```diff
@@ -0,0 +1,18 @@
+package com.baeldung.fastexcel;
+
+import org.dhatim.fastexcel.reader.Cell;
+import org.dhatim.fastexcel.reader.Row;
+
+public class FastexcelHelper {
+	public boolean isRowEmpty(Row row) {
```

Comment:
This is not formatted as per the standard. You're using tabs, where we should be using spaces.